packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -611,6 +614,30 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
 
     await tap(backButton);
   }
+
+  /// Attempts to find the [SemanticsData] of first result from [finder].
```

Comment:
document what happens if the object identified by finder doesn't own its own semantics node?

Reply:
addressed in https://github.com/flutter/flutter/pull/19046/files